tests/performance-tests/src/utils/httpClient.ts

```diff
@@ -0,0 +1,94 @@
+import http, { RefinedParams, RefinedResponse, RequestBody } from 'k6/http';
+
+type HttpHeaders = { [name: string]: string };
```

Comment:
Would suggest using an interface here instead of `type`

Reply:
Done here, and also for all the `types` in dataService.ts too.  Ta.